src/cli.ts

```diff
@@ -5,6 +5,7 @@ import orpcGenerator from "./";
 import { parseOpenRPCDocument } from "@open-rpc/schema-utils-js";
 
 program
+  .version(`v${require('../../package.json').version}`, '-v, --version')
```

Comment:
```suggestion
  .version(`${require('../../package.json').version}`, '-v, --version')
```

as documented in pristine, we never prefix versions with `v`